cognite_toolkit/cdf_tk/templates.py

```diff
@@ -271,6 +277,20 @@ def check_yaml_semantics(parsed: dict | list, filepath_src: Path, filepath_build
     elif resource_type in ["data_sets", "timeseries", "files"] and isinstance(parsed, list):
         ext_id = ""
         ext_id_type = "multiple"
+    elif resource_type in ["functions"] and "schedule" in filepath_src.stem:
```

Comment:
Not in this PR, but this logic should maybe be in the loader.

Reply:
Good idea, I created https://cognitedata.atlassian.net/browse/CDF-20809 to track that.